styleguide/derek/assets/typography.jade

```diff
@@ -0,0 +1,68 @@
+.container.standard-padding
+
+  .row.half-padding-full
+    .col-md-12
+      h1.red.left Typography
+    .col-md-6
+      p Every visual element of our brand says something about us — even down to the fonts we use and how we use them. The specific styles, sizes and weights communicate the essence of the brand and place our brand within a meaningful hierarchy.
+
+  .row.half-padding-full
+    .col-md-12
+      h4.red Heading Usage
+      p H1 - h3 can be used in <code>$darkest-gray</code> and <code>$white</code> only and all caps only. <br/> They all use Open Sans <code>$heading</code>, uppercase <code>.uppercase;</code>, and font-weight of 200.
```

Comment:
Seems a little inconsistent to me using capitalized `H1` and lower case `h3`. I realize its the start of a sentence, but since its referencing a particular tag styling, I'm wondering if it should be lower case.  Thoughts?